src/test/resources/soop_sst_trajectory.xml

```diff
@@ -6,68 +6,849 @@
     <virtualDataTable>select trajectory_id as instance_id, * from measurements</virtualDataTable>
     <virtualInstanceTable>select * from indexed_file</virtualInstanceTable>
   </source>
-
   <filename>
-    <sql>select 'file-'||(select id from $instance)||'.nc'
+    <sql>       
+        select
+        'IMOS_SOOP-SST_'
+        ||to_char(min("TIME" AT TIME ZONE 'UTC'), 'YYYYMMDD"T"HH24MISS"Z"')
+        ||'_'
+        ||platform_code
+        ||'_FV01_'
+        ||'END-'
+        ||to_char(max("TIME" AT TIME ZONE 'UTC'), 'YYYYMMDD"T"HH24MISS"Z"')
+        ||'_id-'
+        ||id
+        ||'.nc'
+        from $data
+        group by platform_code, id    
     </sql>
   </filename>
-
-  <globalattributes/>
-
   <dimensions>
     <dimension name="TIME"/>
   </dimensions>
-  <variables>
+  <globalattributes/>
+    <attribute name="file_version" value="'Level 1 - Quality Controlled Data'"/>
+	<attribute name="file_version_quality_control">
+		<value>'Data in this file has been through the BOM quality control procedure (Reference Table F). Every data point in this file has an associated quality flag.'</value>
+    </attribute>
+    <attribute name="project" value="'Integrated Marine Observing System (IMOS)'"/>
+    <attribute name="Conventions" value="'CF-1.6,IMOS-1.3'"/> 
+    <attribute name="institution" value="'Australian Bureau of Meteorology'"/> 
+    <attribute name="standard_name_vocabulary" value="'CF-1.6'"/>
+    <attribute name="date_created">
+      <sql>select to_char(now() AT TIME ZONE 'UTC', 'YYYY-MM-DD"T"HH24:MI:SS"Z"')</sql>
+    </attribute>
+    <attribute name="comment" sql="select comment from $instance"/>   --????
+    <attribute name="references" value="'http://www.imos.org.au'"/>
+    <attribute name="vessel_name" value="select vessel_name from $instance"/>
+    <attribute name="platform_code" sql="select platform_code from $instance"/>
+    <attribute name="deployment_code" sql="select voyage_number from $instance"/>
```

Comment:
not deployment_code here but maybe cruise or voyage, check IMOS netcdf user manual.